cpp/ql/lib/semmle/code/cpp/exprs/Call.qll

```diff
@@ -300,6 +300,11 @@ class FunctionCall extends Call, @funbindexpr {
     this.isVirtual() or
     this.getTarget().getAnAttribute().getName() = "weak"
   }
+
+  override predicate mayBeFromImplicitlyDeclaredFunction() {
+    this.getType() instanceof IntType and
```

Comment:
How can this be anything else than `int`?

Reply:
I don't think it can be, so perhaps this test is redundant. But, I'm wondering what happens if a function is implicit in some compilation units, but explicit in others?

Reply:
It sound like that's something that need further thought, especially with regard to the implications that has for the changes made here?

Reply:
I've added a test for this scenario, and removed the redundant test. So it seems that `Function::getType()` can return multiple things if there are conflicting declarations.

Reply:
I've added a test for this scenario, and removed the redundant test. So it seems that `Function::getType()` can return multiple things if there are conflicting declarations.